### PR TITLE
Add 2022 demo page to website

### PR DESCRIPTION
**Pull request by melcorr**
* Add Demo 105
* Add link to 2022 to index page
* Add link to index to 2022 page

I previewed the build and all links working etc.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -64,7 +64,7 @@ navigation:
       - title: Events
         url: https://theforeman.org/events
       - title: Community Demos
-        url: /demos/2021.html
+        url: /demos/2022.html
```
Comment (ekohl) [inline _config.yml:67]: I wonder if this should start linking to the demos overview
```suggestion
        url: /demos/
```
That way users can also look back at the 2021 and earlier demos.

File: demos/2022.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+advance on our [Community Events Calendar](https://community.theforeman.org/c/events/l/calendar).
+
+Here is a list of every demo that has taken place this year.
+Our demos are split into two sections: a user-focused section and a developer-focused section.
+
+If you've any questions or feedback, we welcome any feedback in our
+[community](https://community.theforeman.org)
+
+You can find older demos here:
+* [Foreman Community Demos](/demos/index.html)
```
Comment (ekohl) [inline demos/2022.md:23]: You don't need to include `index.html`
```suggestion
* [Foreman Community Demos](/demos/)
```

File: demos/2022.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+---
+layout: demos
+title: Community Demos
```
Comment (ekohl) [inline demos/2022.md:3]: Should include the year to differentiate pages? If so, we should do the same with the 2021 page.
```suggestion
title: Community Demos 2022
```

**Review by ekohl — COMMENTED** [review]
Overall :+1: with some possible improvements inline.

File: _config.yml (hunk 1 above)
Comment (melcorr) [reply _config.yml:67]: In reality, why would anyone want to watch old demos? 
On YouTube, there is a stagnation in views after about a month.
QE etc are interested for a max of about six months. 
I don't want to fetishize this process. This suggestion adds an extra step for users.

Comment (ekohl) [reply _config.yml:67]: I did see (after I placed this comment) you added a link on the 2022 page itself. That is something we previously didn't have and may be sufficient to cover the use case.

**Comment by ekohl** [comment]
Did you close this accidentally? Because I'm ok with merging it even without changes.

**Pull request closed without merging**